themes/classic/templates/catalog/product.tpl

```diff
@@ -183,119 +108,86 @@
-                                    <td>{$quantity_discount.discount}</td>
-                                    <td>{l s='Up to %s' sprintf=$quantity_discount.save}</td>
-                                  </tr>
-                                {/foreach}
-                              </tbody>
-                            </table>
-                          </section>
-                          <hr>
-                        {/if}
-                      {/block}
-
-                      {block name='product_add_to_cart'}
-
-                          {*<form class="add-to-cart" action="{$urls.pages.cart}" method="post">*}
-
-                            {block name='product_quantity'}
-                              <p class="product-quantity _margin-top-medium">
-                                <label for="quantity_wanted">{l s='Quantity'}</label>
-                                <input type="number" min="1" name="qty" id="quantity_wanted" value="{$product.quantity_wanted}" class="input-group" />
-                              </p>
-                            {/block}
-
-                            {block name='product_minimal_quantity'}
-                              {if $product.minimal_quantity > 1}
-                                <p class="product-minimal-quantity">
-                                  {l s='The minimum purchase order quantity for the product is %s.' sprintf=$product.minimal_quantity}
-                                </p>
-                              {/if}
-                            {/block}
-
-                            <button class="btn btn-primary add-to-cart _relative" data-button-action="add-to-cart" type="submit" {if !$product.add_to_cart_url}disabled{/if}>
-                              <i class="material-icons shopping-cart">&#xE547;</i>
-                              {l s='Add to cart'}
-                            </button>
-
-
-                            {block name='product_availability'}
-                             {if $product.show_availability}
-                                <p id="product-availability" class="_margin-left-medium"><i class="material-icons check">&#xE5CA;</i>{$product.availability_message}</p>
-                             {/if}
-                            {/block}
-                            {hook h='displayProductButtons' product=$product}
-                          {*</form>*}
-
-                      {/block}
-
-                      {block name='product_refresh'}
-                        <input class="product-refresh _margin-top-large ps-hidden-by-js" name="refresh" type="submit" value="{l s='Refresh'}" />
-                      {/block}
-                    </div>
+                        {block name='product_availability'}
+                         {if $product.show_availability}
+                            <p id="product-availability" class="_margin-left-medium"><i class="material-icons check">&#xE5CA;</i>{$product.availability_message}</p>
```

Comment:
i must say that .check is pretty generic name for an icon ;-) please consider changing this